air/src/options.rs

```diff
@@ -378,6 +378,9 @@ impl PartitionOptions {
     /// Returns the size of each partition used when committing to the main and auxiliary traces as
     /// well as the constraint evaluation trace.
     pub fn partition_size<E: FieldElement>(&self, num_columns: usize) -> usize {
```

Comment:
Would be good to clarify in the comment that the returned number of columns is the number in the field `E` (rather than the base field of `E`).

Reply:
Agreed, added